src/main/java/uk/gov/hmcts/reform/hmc/model/HearingRequest.java

```diff
@@ -25,4 +25,7 @@
     @Valid
     private List<PartyDetails> partyDetails;
 
+    public RequestDetails getRequestDetails() {
+        return null;
```

Comment:
Least impactful change for now since we no longer need a separate class for the POST, and `RequestDetails` won't be provided anymore for POST since we no longer require services to provide a timestamp.